app/Http/Controllers/NotificationsController.php

```diff
@@ -173,4 +183,48 @@ private function endpointUrl()
 
         return $url;
     }
+
+    private function markReadByIds($ids)
+    {
+        $user = auth()->user();
+        if ($user->userNotifications()->whereIn('notification_id', $ids)->update(['is_read' => true])) {
+            event(new NotificationReadEvent($user->getKey(), $ids));
```

Comment:
might as well pass `['ids'=>$ids]` for `$params` and remove the second argument?